uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Validator/Date.php

```diff
@@ -21,13 +19,10 @@ class CRM_HRLeaveAndAbsences_Validator_Date {
    *
    * @return bool
    */
-  public static function isValid($date, $format = 'Y-m-d')
+  public static function isValid($date, $format = 'YmdHis')
```

Comment:
Sorry I didn't notice this before, but this actually is changing the class/method behavior. This class is just supposed to validate dates but with this its now validating datetimes. Maybe we should rename the class? Although I think having two separated classes would be better?